src/index.mjs

```diff
@@ -0,0 +1,40 @@
+import {GrafanaClient} from "./grafana-client.mjs";
+import {BackupStorage} from "./backup-storage.mjs";
+import {getInput} from "./utils.mjs";
+
+const GRAFANA_URL = getInput('grafana-url');
+const API_TOKEN = getInput('api-key');
+const OUTPUT_DIR = getInput('dir', 'dashboards');
+
+if (!GRAFANA_URL || !API_TOKEN) {
+  console.error('Error: GRAFANA_URL and API_TOKEN must be set!');
+  process.exit(1);
+}
```

Comment:
`getInput` throws an error if the input is not provided. Does it make sense to have this check?